_pages/introduction.md

```diff
@@ -0,0 +1,124 @@
+
+### Where regulations are published
+
+The Government Printing Office publishes an official printed paper Code of Federal Regulations and [official digital Code of Federal Regulations (on FDsys)](https://www.gpo.gov/fdsys/browse/collectionCfr.action?collectionCode=CFR) once a year.
+
+Throughout the year, GPO publishes changes to regulations through [e-CFR (Electronic Code of Federal Regulations)](http://www.ecfr.gov/), though this is not legally binding.
+
+Proposed changes and final changes effective some time in the future are published in the [Federal Register](https://www.federalregister.gov/) (the “Daily Journal of the United States Government”).
+
+## <a name="how-regulations-change"></a>How regulations change (through “rules”)
+
+For most changes (“rules”), an agency must propose the alterations and request the general public (including special interest groups, industry, etc.) provide feedback. The proposal includes a specific date by which all feedback must be received, though this can sometimes be modified. After the agency receives feedback, the agency must review each unique comment; they may draft a “final” rule which incorporates this feedback, make a different proposal, or withdraw the proposal. If they produce a final rule, they give it an effective date, at which point the changes are applied to the regulation. This process is referred to as “notice and comment”.
+
+Regulations.gov provides [a summary of the steps of the regulatory (rulemaking) process](http://www.regulations.gov/#!home;tab=learn), the Department of Transportation provides [a more in-depth overview of the process](https://www.transportation.gov/regulations/rulemaking-process), and Reginfo.gov provides [a detailed chart of the process (the "Reg Map")](http://www.reginfo.gov/public/reginfo/Regmap/index.jsp).
+
+There are a few different types of Federal Register Notices that can be part of a notice and comment process:
+
+### Advanced Notice of Proposed Rulemaking (ANPR)
+
+This doesn’t happen as often as the following notices. This usually happens before any change is made to a regulation. An ANPR is put out when an agency wants to check if an idea for future regulation change is something the public may want. It is meant to get early feedback from the public, and the issues in this notice may not move forward to the next notice depending on the public’s response.
+
+### Notice of Proposed Rulemaking (NPR) or Proposed Rule
+
+This is the initial draft of a change to a regulation. The notice starts with a “Preamble” which can contain the reasons for the regulation change, some section-by-section analysis, a cost analysis, and other general explanations. The Preamble is followed by the changes to the regulation. This notice only shows the actual changed parts of the regulation, not the new full text of the regulation.
```

Comment:
> This notice only shows the actual changed parts of the regulation, not the new full text of the regulation.

This is a little confusing and perhaps misleading -- for the most part, proposals don't show _any_ part of the original regulation; they just show the additions. Maybe something more general like, "As the notice does not contain the full text of the new regulation, you will likely need to pull up the previous version of the regulation to make sense of the changes"